src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -1327,7 +1330,7 @@ public static void createBucketWithLifeCycle() throws IOException, URISyntaxExce
   @Before(order = 1, value = "@DATASTORE_SOURCE_ENTITY")
   public static void createEntityInCloudDataStore() throws IOException, URISyntaxException {
     kindName = "cdf-test-" + UUID.randomUUID().toString().substring(0, 8);
-   String entityName = DataStoreClient.createKind(kindName);
+    String entityName = DataStoreClient.createKind(kindName);
```

Comment:
nit: checkstyle

Reply:
After reformatting the code this line is the only one which was not formatted correctly. So that's why this change is reflected here.